advanced/Scripts/piholeDebug.sh

```diff
@@ -422,6 +422,36 @@ diagnose_operating_system() {
   fi
 }
 
+check_selinux() {
+  # SELinux is not supported by the Pi-hole
+  echo_current_diagnostic "SELinux"
+  # Check if a SELinux configuration file exists
+  if [[ -f /etc/selinux/config ]]; then
+    # If a SELinux configuration file was found, check the default SELinux mode.
+    DEFAULT_SELINUX=$(egrep -i '^SELINUX=' /etc/selinux/config | cut -d'=' -f2 | awk '{print tolower($0)}')
+    case $DEFAULT_SELINUX in
+      enforcing)
+        log_write "${CROSS} ${COL_LIGHT_RED}Default SELinux: $DEFAULT_SELINUX${COL_NC}"
```

Comment:
`${COL_LIGHT_RED}` and `${COL_LIGHT_GREEN}` [are deprecated](https://github.com/pi-hole/pi-hole/blob/bea99abd8dc9b2459a6ba23396ee59f62293a79a/advanced/Scripts/COL_TABLE#L31), so I'd recommend using `${COL_RED}` and `${COL_GREEN}` instead :smile: